src/components/forms/validation-message.jsx

```diff
@@ -5,13 +5,24 @@ const React = require('react');
 require('./validation-message.scss');
 
 const ValidationMessage = props => (
-    <div className={classNames(['validation-message', props.className])}>
+    <div
+        className={classNames(
+            'validation-message',
+            {
+                'validation-error': props.isError,
+                'validation-ok': props.isOk
+            },
+            props.className
+        )}
+    >
         {props.message}
     </div>
 );
 
 ValidationMessage.propTypes = {
     className: PropTypes.string,
+    isError: PropTypes.bool,
```

Comment:
Another way to do it would be to set a string like `mode` or something to `"error"` or "`ok`". Would that be better?

Reply:
I tried this out, take a look. Also changed "ok" to "info".

Reply:
Hmm, so this is just to get a different classname right? Could that just be passed in without changing the component at all since it is already a prop?

Reply:
Here's my thinking:

I don't like to have components take their own classnames as props, because that seems to me to violate separation of concerns. If I want to use a <ValidationMessage>, my job is to know whether I want one that's in an error state, or an ok/info state. But it's not my job to keep track of which classes it itself includes. And I don't want to pass in my own class's className, because then I have to be responsible for the quirks of how it uses formatting.

Reply:
Sounds good! Let's leave it as you have it. Could you add a test for the new behavior in ValidationMessage and formik-input?

Reply:
Good idea, done!